src/main/java/org/jenkinsci/remoting/engine/JnlpAgentEndpointResolver.java

```diff
@@ -141,7 +160,7 @@ public void setTunnel(String tunnel) {
     public JnlpAgentEndpoint resolve() throws IOException {
         IOException firstError = null;
         for (String jenkinsUrl : jenkinsUrls) {
-            
```

Comment:
Please do not do unrelated whitespace changes. It may cause undesired merge conflicts